### PR TITLE
Tutorials fix

### PR DESCRIPTION
**Pull request by fjolidon**
Small changes to fix a few things I noticed while taking the tutorials.

### CODE REVIEW

**Comment by fjolidon** [comment]
:+1:

File: guide/tutorial-6-4-wheels-robot.md [hunk 1]
```diff
@@ -167,11 +167,9 @@ outside the robot is necessary (see the
 
 > **Hands on**:
 Add the two distance sensors as explained above. The distance sensors are at an
-angle to 0.3 [rad] with the robot front vector. Set their `type` field to
-"sonar". Set their graphical and physical shape to a cube (not transformed)
-having a edge of *0.01* [m]. Set their color to blue. Set their `name` field
-according to the labels of [this
-figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).
+angle to 0.3 [rad] with the robot front vector. Set their graphical and physical
```
Comment (stefaniapedrazzi) [inline guide/tutorial-6-4-wheels-robot.md]: It is fine to remove the instructions to set the DistanceSensor type.
But the sample world in the `webots` repository has to be updated accordingly.
I will take care of this.

File: guide/tutorial-4-more-about-controllers-20-minutes.md [hunk 2]
```diff
@@ -198,11 +198,11 @@ threshold) as follows:
 
 > ```c
 > // detect obstacles
-> bool left_obstacle =
+> bool right_obstacle =
```
Comment (stefaniapedrazzi) [inline guide/tutorial-4-more-about-controllers-20-minutes.md]: These changes have to be ported to the corresponding controller in `webots` repository.
I can take care of this.

**Comment by stefaniapedrazzi** [comment]
This PR still has to be merged

**Pull request merged**